cfme/storage/volume.py

```diff
@@ -165,6 +165,23 @@ class VolumeSnapshotView(VolumeView):
     cancel = Button('Cancel')
 
 
+class AttachInstanceView(VolumeView):
+    is_displayed = displayed_not_implemented
```

Comment:
Question:
can you please explain to me why we should not write `is_displayed` property? I think we are getting `title` and `instance` dropdown to check

Reply:
Actually you are right. I just thoght is is like some methods where is_displayed is not implemented. working on that !